docs/troubleshooting-guide/data-permissions.md

```diff
@@ -0,0 +1,122 @@
+2. If the person is in multiple groups:
+    - Remove the person from the group with wider permissions, or
+    - Go to **Admin** > **Permissions** and change the **Data access** permission type.
+
+**Explanation** 
+
+If a person is a member of multiple [groups][groups], Metabase will grant the _most permissive_ level of access across all the groups in which they're a member.
+
+For example, if a person is a member of two groups — one which grants [Unrestricted][unrestricted] access to a database or table and another that grants [No self-service][no-self-service] access — that person will have full unrestricted access.
+
+Remember that everyone is a member of the **All Users** group. We recommend you revoke permissions from the **All users** group, and create new groups to selectively apply permissions to your data sources.
+
+## People can view collections that contain restricted data
+
+{% include plans-blockquote.html %}
+
+1. Go to **Admin** > **Permissions** and select the user group.
+2. Select the database or table that you want to restrict.
+3. Choose **Block** from the dropdown.
+4. Click **Save**.
+
+**Explanation**
+
+If you've granted a user group **No self-service** access to a database or table, people can still view saved questions and dashboards that draw on that database, as long as those questions and dashboards are stored in a collection that they have access to.
+
+The [**Block** data permission][block-data-permission] is available on [paid versions of Metabase](https://www.metabase.com/pricing). If you block a group's access to a database, members of that group will not be able to see any data from this database, regardless of their collection permissions.
+
+## Getting a "permission denied" error message
+
+If you get an error message that says something like "permission denied to \<your table\>", you'll need to check if the Metabase application has the correct permissions to query your database.
+
+1. Go to the SQL editor and run a basic query against the table or schema in question:
+    ```
+    SELECT 1
+    FROM <your table>;
+    ```
+2. Get the credentials that Metabase uses to connect to your database. Ask your database admin if you're not sure.
+3. Run the query from step 1 using a different application (such as your CLI or database IDE) and your Metabase's database credentials.
+4. If you cannot access the table or schema in both step 1 and 3, ask your database admin to:
+    - Grant permissions to the role that Metabase is using to connect, or
+    - Provide a set of database credentials with the correct permissions.
+
+**Explanation** 
+
+Your database has its own set of permissions that are configured for each person (or application) that logs in. The database permissions can block Metabase from connecting to certain schemas or tables, regardless of the permissions you've set up on the Metabase side.
```

Comment:
Can you rephrase this paragraph? It's relatively clear, but what's lost is the emphasis on the fact that it's the connection credentials (the database user account Metabase logs in to the database with) that determines what Metabase can see _before_ any groups and permissions are set up in Metabase.